workers/wfsfeature-harvester/pom.xml

```diff
@@ -32,7 +32,7 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>wfsfeature-harvester</artifactId>
+  <artifactId>gn-wfsfeature-harvester</artifactId>
```

Comment:
Parent artifactId should be `gn-workers` instead of `workers`
```xml
  <parent>
    <artifactId>gn-workers</artifactId>
    <groupId>org.geonetwork-opensource</groupId>
    <version>3.9.0-SNAPSHOT</version>
  </parent>
```